features/coffeescript_compilation_error.feature

```diff
@@ -3,6 +3,12 @@ Feature: Coffeescript compilation error
   As a developer with an error in my coffeescript file
   I want an appropriate error message
 
+  Background:
+    Given I have configured "filePattern" to be "**/*.coffee"
+    And I have configured "transpilers" to contain
+      | extension | module        |
```

Comment:
Uppercase table headers to distinguish them from the rest? I know this isn't specified in Cucumber, but I really miss it when its not there and think Cucumber tables are much better off with this convention. There are legitimate tables that dont